homeassistant/components/tradfri/sensor.py

```diff
@@ -48,18 +63,41 @@ def _get_air_quality(device: Device) -> int | None:
     return cast(int, device.air_purifier_control.air_purifiers[0].air_quality)
 
 
-SENSOR_DESCRIPTION_AQI = TradfriSensorEntityDescription(
-    device_class=SensorDeviceClass.AQI,
-    native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
-    key=SensorDeviceClass.AQI,
-    value=_get_air_quality,
+def _get_filter_time_left(device: Device) -> int:
+    """Fetch the filter's remaining life (in hours)."""
+    return cast(
+        int, device.air_purifier_control.air_purifiers[0].filter_lifetime_remaining
+    )
+
+
+SENSOR_DESCRIPTIONS_BATTERY: tuple[TradfriSensorEntityDescription, ...] = (
+    TradfriSensorEntityDescription(
+        device_class=SensorDeviceClass.BATTERY,
+        native_unit_of_measurement=PERCENTAGE,
+        key=SensorDeviceClass.BATTERY,
+        value=lambda device: cast(int, device.device_info.battery_level),
+    ),
 )
 
-SENSOR_DESCRIPTION_BATTERY = TradfriSensorEntityDescription(
-    device_class=SensorDeviceClass.BATTERY,
-    native_unit_of_measurement=PERCENTAGE,
-    key=SensorDeviceClass.BATTERY,
-    value=lambda device: cast(int, device.device_info.battery_level),
+
+SENSOR_DESCRIPTIONS_FAN: tuple[TradfriSensorEntityDescription, ...] = (
+    TradfriSensorEntityDescription(
+        key=SensorDeviceClass.AQI,
```

Comment:
I don't think we should use the device class for key. We should define integration specific keys that only we control.

Reply:
I've changed the keys.